lib/absinthe/phase/subscription/subscribe_self.ex

```diff
@@ -31,7 +31,7 @@ defmodule Absinthe.Phase.Subscription.SubscribeSelf do
       {:replace, blueprint,
        [
          {Phase.Subscription.Result, topic: subscription_id},
-         {Phase.Telemetry, [:execute, :operation, :stop, options]}
+         {Phase.Telemetry, Keyword.put(options, :event, [:execute, :operation, :stop])}
```

Comment:
@binaryseed note this change. This is partially to appease Dialyzer since the spec for phase config says that these are keywords. It's also handy because it allows us to more consistently push out telemetry events with this info.

Reply:
makes sense!